streams/src/main/java/org/apache/kafka/streams/processor/internals/SinkNode.java

```diff
@@ -63,7 +63,7 @@ public void init(ProcessorContext context) {
         if (this.valSerializer instanceof ChangedSerializer &&
                 ((ChangedSerializer) this.valSerializer).inner() == null)
             ((ChangedSerializer) this.valSerializer).setInner(context.valueSerde().serializer());
-
+        this.nodeMetrics = new NodeMetrics(context.metrics(), name(),  "task." + context.taskId());
```

Comment:
In `ProcessorNode` we also do

> this.nodeMetrics.metrics.recordLatency(nodeMetrics.nodeCreationSensor, startNs, time.nanoseconds());

should we do the same here? What about a more general way to measure this latency? It seems we do not do this correctly right -- we could abstract `nodeMetrics` such that it can be private in and avoid code duplication.